clusters/build-clusters/build10/multiarch-manager-operator/admin_30-rolebinding.yaml

```diff
@@ -4,21 +4,21 @@ items:
   - kind: RoleBinding
     apiVersion: rbac.authorization.k8s.io/v1
     metadata:
-      name: openshift-multiarch-manager-operator-mmo-devel
-      namespace: openshift-multiarch-manager-operator
+      name: openshift-multiarch-tuning-operator-mto-devel
+      namespace: openshift-multiarch-tuning-operator
     roleRef:
       apiGroup: rbac.authorization.k8s.io
       kind: ClusterRole
       name: admin
     subjects:
       - apiGroup: rbac.authorization.k8s.io
         kind: Group
-        name: mmo-devel
+        name: mto-devel
   - kind: RoleBinding
     apiVersion: rbac.authorization.k8s.io/v1
     metadata:
-      name: mmo-devel-test
-      namespace: mmo-devel-test
+      name: mto-devel-test
+      namespace: mto-devel-test
```

Comment:
https://github.com/openshift/release/blob/92c7c4cf37fa44aa0f0d08987be11355e03b483d/clusters/build-clusters/build10/multiarch-manager-operator/admin_20-test-namespace.yaml

Reply:
^ needs a change in name too.